hack/gen-versions/components.go

```diff
@@ -36,6 +36,7 @@ var defaultImages = map[string]string{
 	"eck-kibana":              "tigera/kibana",
 	"guardian":                "tigera/guardian",
 	"tigera-cni":              "tigera/cni",
+	"rdtigera/init-container": "rdtigera/init-container",
```

Comment:
I think the key in this map is just a name for the image, so can be `tigera/init-container`.

Also, I think we should change the name of this container. `init-container` is a very vague term and doesn't describe what the container does. The container should be called something like `tigera/cert-requester` or something like that.

Reply:
We discussed this offline, a follow-up will change this to tigera/key-cert-provisioner